src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -78,21 +108,7 @@ public static class OpenApiService
                 predicate = OpenApiFilterService.CreatePredicate(requestUrls: requestUrls, source:document);
                 document = OpenApiFilterService.CreateFilteredDocument(document, predicate);
             }
-
-            var context = result.OpenApiDiagnostic;
-
-            if (context.Errors.Count > 0)
-            {
-                var errorReport = new StringBuilder();
-
-                foreach (var error in context.Errors)
-                {
-                    errorReport.AppendLine(error.ToString());
-                }
-
-                throw new ArgumentException(string.Join(Environment.NewLine, context.Errors.Select(e => e.Message).ToArray()));
-            }
-
+            
```

Comment:
on line 122 result will be null if the doc type is csdl.

Reply:
Resolving..